docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_RESTful_Web_Services.adoc

```diff
@@ -114,4 +114,6 @@ or disable RESTEasy providers. Note that they can be used in conjunction with
 MicroProfile REST Client client running inside a JAX-RS resource, in which the embedded javax.ws.rs.core.Response is
 "sanitized" before being returned to prevent the risk of information leaking from a third party. The original
 behavior can be restored by setting the parameter "resteasy.original.webapplicationexception.behavior" to "true".
-See the RESTEasy User Guide chapter "Resteasy WebApplicationExceptions" for more information.
\ No newline at end of file
+See the RESTEasy User Guide chapter "Resteasy WebApplicationExceptions" for more information.
+
+NOTE: References in this document to Java API for RESTful Web Services(JAX-RS) refer to Jakarta RESTful Web Services unless otherwise noted.
```

Comment:
JAX-RS appears three more times in this file.

Reply:
I'm following this document - https://jakarta.ee/legal/acronym_guidelines/ 
where it has mentioned -
`If a specification document (or related materials) contains references to an Oracle/JCP acronym, include a sentence such as the following: “References in this document to JMS refer to the Jakarta Message Service unless otherwise noted." The sentence needs to be a legible size and should appear early in the specification document (or related materials), not in an illegibly small footnote. A reader should not have to scroll, turn a page, or click a link to get to the sentence.`
That's why I have mentioned the note at the end of the file where a many times repeated replacement of 'JAX-RS' with the much more verbose 'Jakarta RESTful Web Services' is pretty clunky.

Reply:
> That's why I have mentioned the note at the end of the file where

Oh, OK, I didn't know that. Thanks.

>  reader should not have to scroll, turn a page

Given that, would it be better to put the note earlier than at the end of the file?

Reply:
Yes, sure! I'll adjust that :)